python/py-isce2/Portfile

```diff
@@ -0,0 +1,154 @@
+
+1) Set environment variable ISCE_HOME=${python.pkgd}/isce .
+2) Append \$ISCE_HOME/applications and \$ISCE_HOME/bin to PATH.
+3) Stack processing scripts can be found under ${prefix}/share/${subport}/stack
+
+
+Other important notes
+----------------------
+
+1) GDAL is required for working with Radarsat2, Terrasar-X and Sentinel 1-A. Suggested variant of gdal is  +expat+geos+hdf5+netcdf+postgresql95+sqlite3
+
+2) One can use isce2 with other packages like SpiceyPy, sarpy etc. These will need to be installed from source separately.
+
+    "
+
+    depends_lib-append      port:curl \
+                            port:bzip2\
+                            port:openmotif\
+                            port:hdf5 \
+                            port:fftw-3\
+                            port:fftw-3-single\
+                            port:gdal \
+                            port:scons\
+                            port:py${python.version}-setuptools\
+                            port:py${python.version}-cython\
+                            port:py${python.version}-numpy \
+                            port:py${python.version}-scipy \
+                            port:py${python.version}-nose  \
+                            port:py${python.version}-gdal  \
+                            port:py${python.version}-h5py
+
+    require_active_variants port:curl ssl
+    require_active_variants port:gdal hdf5
+    require_active_variants port:gdal netcdf
+
+    post-patch {
+        system "find ${workpath} -type f -print0 | xargs -0 -L1 env LANG=C sed -i '' 's|cython3|${prefix}/bin/cython-${python.branch}|g'"
+    }
+
+    pre-configure {
+        xinstall -d ${workconfpath}
+        xinstall -d ${workbldpath}
+    }
+
+    configure {
+        #Check if compiler is set
+        if {![gcc_variant_isset]} {
+            default_variants-append +gcc7
+        }
+
+        #For gcc, set libstdc++
+        configure.cxx_stdlib libstdc++
+
+		#Setup SConfigISCE and config dir here
+        set gccvar  [gcc_variant_name]
+        set gcclast [string index ${gccvar} end]
+        set cfgname ${workconfpath}/SConfigISCE
+        set fileId [open $cfgname "w"]
+
+
+        puts   $fileId  "PRJ_SCONS_BUILD  =   ${workbldpath}"
+        puts   $fileId  "PRJ_SCONS_INSTALL =  ${workinstallpath}"
+        puts   $fileId  "LIBPATH  =      ${prefix}/lib"
+        puts   $fileId  "CPPPATH  =      ${python.include}"
+        puts   $fileId  "CC       =      ${prefix}/bin/gcc-mp-${gcclast}"
+        puts   $fileId  "CXX      =      ${prefix}/bin/g++-mp-${gcclast}"
+        puts   $fileId  "FORTRANPATH  =   ${prefix}/include"
+        puts   $fileId  "FORTRAN   =     ${prefix}/bin/gfortran-mp-${gcclast}"
+        puts   $fileId  "MOTIFLIBPATH =  ${prefix}/lib"
+        puts   $fileId  "X11LIBPATH   =  ${prefix}/lib"
+        puts   $fileId  "MOTIFINCPATH =  ${prefix}/include"
+        puts   $fileId  "X11INCPATH   =  ${prefix}/include"
+        close   $fileId
```

Comment:
I wonder if this should probably be handled with a patchfile and substitution.

Reply:
The patchfile will be for a completely new file that doesnt patch an existing file. Is that ok?

Reply:
It's absolutely okay.